src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -193,17 +296,18 @@ protected override void Free()
             BoundNode node,
             ArrayBuilder<(BoundReturnStatement, TypeSymbolWithAnnotations)> returnTypesOpt,
             VariableState initialState,
-            Action<BoundExpression, TypeSymbolWithAnnotations> callbackOpt)
+            PooledDictionary<BoundExpression, (NullabilityInfo, TypeSymbol)> analyzedNullabilityMapOpt = null)
```

Comment:
> = null [](start = 102, length = 7)

Does this need to be optional?

Reply:
Yes, the `BestTypeForLambdaReturns` path uses this. Again, I can make that pass an explicit value if you would prefer.

---
In reply to: [265688501](https://github.com/dotnet/roslyn/pull/34017#discussion_r265688501) [](ancestors = 265688501)